crates/cxx-qt-gen/src/writer/rust/mod.rs

```diff
@@ -178,12 +181,20 @@ pub fn write_rust(generated: &GeneratedRustBlocks) -> TokenStream {
         // Add the type alias to the C++ struct
         let cpp_struct_ident = &qobject.cpp_struct_ident;
         let rust_struct_ident = &qobject.rust_struct_ident;
-        qobject_types.push(quote! { pub type #rust_struct_ident = super::#cpp_struct_ident; })
+        let rust_struct_ident_str = rust_struct_ident.to_string();
+        qobject_types.push(quote! {
+            #[doc = "The C++ type for the QObject "]
```

Comment:
I think this comment could be more extensive and explain the relationship between the C++ QObject and the Rust struct.
Maybe just link to the book?

Reply:
Done